pallets/capacity/src/lib.rs

```diff
@@ -579,7 +579,7 @@ impl<T: Config> Pallet<T> {
 			CurrentEpochInfo::<T>::set(EpochInfo { epoch_start: current_block });
 			T::WeightInfo::on_initialize()
 				.saturating_add(RocksDbWeight::get().reads(1))
-				.saturating_add(RocksDbWeight::get().writes(1))
+				.saturating_add(RocksDbWeight::get().writes(2))
```

Comment:
@enddynayn This isn't really from this PR at all, but I think we want to use `T::DbWeight` instead of the older `RocksDbWeight` in prep for the eventual switch to ParityDB. (Likely same in the FrequencyTxPayment pallet as well.). Feel free to do in this PR or create a new issue for it.



```suggestion
				.saturating_add(T::DbWeight::get().writes(2))
```